src/Generators/TestsGenerator.php

```diff
@@ -93,49 +104,36 @@ protected function getDumpValuesList($model)
     {
         $values = $this->buildEntityObject($model);
 
-        return array_associate($values, function ($value, $key) {
+        array_walk($values, function (&$value) {
             if ($value instanceof \DateTime) {
```

Comment:
let's move DateTime to use block

```suggestion
            if ($value instanceof DateTime) {
```

Reply:
Done